src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -31,27 +31,14 @@ public void isSamePerson() {
         // null -> returns false
         assertFalse(ALICE.isSamePerson(null));
 
-        // different phone and email -> returns false
-        Person editedAlice = new PersonBuilder(ALICE).withPhone(VALID_PHONE_BOB).withEmail(VALID_EMAIL_BOB).build();
-        assertFalse(ALICE.isSamePerson(editedAlice));
+        // same name, different attributes -> returns true
```

Comment:
nit, to be absolutely clear:
`//same name, all other attributes different -> returns true`